website/templates/index.jsx

```diff
@@ -1,15 +1,38 @@
 import React from 'react';
 import {Home} from 'gatsby-theme-ocular/components';
-import './style.css';
+import GLTFExample from './example-gltf';
+import styled from 'styled-components';
 
 if (typeof window !== 'undefined') {
   window.website = true;
 }
 
-const HeroExample = require('./example-gltf').default;
+const Bullet = styled.li`
+  background: url(images/icon-high-precision.svg) no-repeat left top;
+  list-style: none;
+  max-width: 540px;
+  padding: 8px 0 12px 42px;
+  font: ${props => props.theme.typography.font300};
+`;
+
+const HeroExample = () => <GLTFExample panel={false} />
 
 export default class IndexPage extends React.Component {
   render() {
-    return <Home HeroExample={HeroExample} />;
+    return (
+      <Home HeroExample={HeroExample} >
+        <ul>
+          <Bullet>
+            Parsers and encoders for many major 3D, geospatial and tabular formats.
```

Comment:
I like the JSX tags, though these read a weirdly without their titles. Also would be cool to keep the ability to use the icons.